i6_models/parts/conformer/mhsa.py

```diff
@@ -10,14 +10,17 @@
 
 @dataclass
 class ConformerMHSAV1Config(ModelConfiguration):
+    """
+    :param input_dim: input dim and total dimension for query/key and value projections, should be divisible by `num_att_heads`
+    :param num_att_heads: number of attention heads
+    :param att_weights_dropout: attention weights dropout
+    :param dropout: multi-headed self attention output dropout
+    """
+
     input_dim: int
-    """input dim and total dimension for query/key and value projections, should be divisible by `num_att_heads`"""
```

Comment:
Why this is changed now? i think the proper documentation for dataclass attributes is either this way or something like this:
```
input_dim: int  # ...
```
or did we agree somehwere that we want to use this?

Reply:
>As far as I have seen, PyCharm supports normal Class-based param inits now also for dataclasses, so there is no need anymore to have this specific docstring style. But this can be adapted later if wanted.

Is what @JackTemaki said. And from what I saw the doc string came up this way in PyCharm.

Reply:
It does not for me?

<img width="402" alt="image" src="https://github.com/rwth-i6/i6_models/assets/59132/d2961e0d-f7f4-457d-8f21-0a9837a9962c">

However, the docstring works for this standard variant we had before here.

Reply:
Same for me.

Reply:
I'd prefer the variant that @Atticus1806 proposed.

Reply:
In my opinion this makes the code more messy because it inreases the lines the code block has. Hovering over the attribute in the code still gives you the doc string.
In the end I don't think this is important, we should just take a decision I can implement so this PR does not take too much longer.